static/js/controllers/reports.js

```diff
@@ -250,17 +335,18 @@ var _ = require('underscore'),
             _query();
           }
         }
-
       };
 
       $scope.$on('ClearSelected', function() {
-        $scope.selected = null;
+        $scope.selected = [];
+        $('#reports-list input[type="checkbox"]')
+          .prop('checked', false);
         liveList.clearSelected();
```

Comment:
The livelist clearSelected method is type agnostic so I didn't want to put report specific code in there. When other tabs have a bulk delete/edit option this would make sense.